parrots.json

```diff
@@ -674,5 +674,10 @@
       "gif": "databaseparrot.gif",
       "name": "Database Parrot",
       "tip": "To sql or not to sql"
+    },
+    {
+      "gif": "hd/phparrot.gif",
```

Comment:
Looks like this key actually needs to be `"hd": "hd/phpparrot.gif"` to get tests passing and the build working.

Reply:
Oh, sorry, fixed